### ISSUE TITLE
[Feature Request] Allow to Filter Searched Videos 

### ISSUE THREAD

**Issue by switchtegrax1** [TeamNewPipe/NewPipe#1157]
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I checked if the issue/feature exists in the latest version.

A basic video filtering option would be a great addition to newpipe

**Comment by theScrabi** [comment]
Yes we need to consider this, however i think this is a duplicate.

**Comment by my-password-is-password** [comment]
Watching live streams on NewPipe is awesome but I would love to be able to filter search results by "Live" and "View Count" just like on youtube.  Right now if want to find "Live" streams its all scattered with video clips.

**Youtube search results filtered:**

![youtubesort](https://user-images.githubusercontent.com/6408935/38472957-a11a0132-3b3c-11e8-91b0-6c1152a767be.png)

**NewPipe search results:**

![newpipe](https://user-images.githubusercontent.com/6408935/38472979-df5876c2-3b3c-11e8-8fe6-86f05b7f2791.png)

**Comment by justanidea** [comment]
I post it here :  could we add the option "search in subscriptions"? Because with a lot of channels its very long to scroll down for the letter "Z" :)

**Comment by theScrabi** [comment]
Yea we definetly need filters, however that requres the backend to be refactored.

**Comment by Stypox** [comment]
Duplicate of #110
